ironfish/src/workerPool/pool.ts

```diff
@@ -122,6 +122,10 @@ export class WorkerPool {
     await Promise.all(workers.map((w) => w.stop()))
   }
 
+  abortJobs(): void {
+    this.workers.forEach((worker) => worker.jobs.forEach((job) => job.abort()))
```

Comment:
It's possible this change doesn't do anything. Did you test that btw?

I saw some docs in node that say worker messages don't be processed if the event loop doesn't yield. We might need to introduce a yield in the miner event loop. I would be curious if the abort line in the actual mine job ever gets triggered.

Reply:
you are right. it does not do anything. I will abort the PR to look into the alternatives.